stixpy/timeseries/quicklook.py

```diff
@@ -139,44 +123,38 @@ def _parse_file(cls, filepath):
 
     @classmethod
     def _parse_hdus(cls, hdulist):
-        """
+        r"""
         Parses STIX FITS data files to create TimeSeries.
 
         Parameters
         ----------
         hdulist :
         """
         header = hdulist[0].header
-        control = QTable(hdulist[1].data)
-        header["control"] = control
-        data = QTable(hdulist[2].data)
-        energies = QTable(hdulist[4].data)
-        energy_delta = energies["e_high"] - energies["e_low"] << u.keV
+        # control = _hdu_to_qtable(hdulist[1])
+        data = _hdu_to_qtable(hdulist[2])
+        energies = _hdu_to_qtable(hdulist[4])
+        energy_delta = energies["e_high"] - energies["e_low"]
 
-        live_time = _lfrac(data["triggers"].reshape(-1) / (16 * data["timedel"] * u.s))
+        live_frac, *_ = get_livetime_fraction(data["triggers"].reshape(-1) / (16 * data["timedel"]))
 
-        data["counts"] = data["counts"] / (live_time.reshape(-1, 1) * energy_delta)
+        data["counts"] = data["counts"] / ((data['timedel'].to(u.s) * live_frac).reshape(-1, 1) * energy_delta)
```

Comment:
@DanRyanIrish this is the fix before was not multiplying the integration time by the livefrac e.g. `live_time.reshape(-1, 1)` -> `(data['timedel'].to(u.s) * live_frac).reshape(-1, 1)`